tests/components/group/test_sensor.py

```diff
@@ -245,16 +245,58 @@ async def test_reload(hass: HomeAssistant) -> None:
     assert hass.states.get("sensor.second_test")
 
 
+async def test_sensor_incorrect_state_with_ignore_non_numeric(
+    hass: HomeAssistant, caplog: pytest.LogCaptureFixture
+) -> None:
+    """Test that non numeric values are ignored in a group."""
+    config = {
+        SENSOR_DOMAIN: {
+            "platform": GROUP_DOMAIN,
+            "name": "test_ignore_non_numeric",
+            "type": "max",
+            "ignore_non_numeric": True,
+            "entities": ["sensor.test_1", "sensor.test_2", "sensor.test_3"],
+            "unique_id": "very_unique_id",
+            "state_class": SensorStateClass.MEASUREMENT,
+        }
+    }
+
+    assert await async_setup_component(hass, "sensor", config)
+    await hass.async_block_till_done()
+
+    entity_ids = config["sensor"]["entities"]
+
+    # Check that the final sensor value ignores the non numeric input
+    for entity_id, value in dict(zip(entity_ids, VALUES_ERROR)).items():
+        hass.states.async_set(entity_id, value)
+        await hass.async_block_till_done()
+
+    state = hass.states.get("sensor.test_ignore_non_numeric")
+    assert state.state == "17.0"
+    assert (
+        "Unable to use state. Only numerical states are supported, entity sensor.test_2 with value string excluded from calculation"
```

Comment:
```suggestion
        "Unable to use state. Only numerical states are supported,"
```
Let's be less explicit so we ensure to capture